compal/models.py

```diff
@@ -33,6 +33,27 @@ class RadioSettings:
     radio_5g: Optional[BandSetting] = None
 
 
+@dataclass
+class InterfaceGuestNetworkSettings:
+    # read-only fields
+    radio: str
+    guest_mac: str
+    # editable-fields
+    enable: Optional[int] = None
+    ssid: Optional[str] = None
+    hidden: Optional[int] = None
+    re_key: Optional[int] = None
+    security: Optional[int] = None
+    pre_shared_key: Optional[str] = None
+    wpa_algorithm: Optional[int] = None
+
+
+@dataclass
+class GuestNetworkSettings:
+    guest_networks_2g: list[InterfaceGuestNetworkSettings] = None
```

Comment:
I think these need to be `List` to fix the warnings,
```suggestion
    guest_networks_2g: List[InterfaceGuestNetworkSettings] = None
```

Line 4 also needs to import it, such as with `from typing import List, Optional`

Reply:
I checked it: If you change this the tests probably pass!

Reply:
Thanks for this hint. 
I changed it accordingly - hope the tests will not fail anymore.